### ISSUE TITLE
在进行OAT量化时，教师模型的权重teacher_model_path ./assets/yolov6s_v2_reopt_43.1.pt ，这个是哪一步得出的结果？

### ISSUE THREAD

**Issue by lanRHC** [meituan/YOLOv6#795]
### Before Asking

- [X] I have read the [README](https://github.com/meituan/YOLOv6/blob/main/README.md) carefully. 我已经仔细阅读了README上的操作指引。

- [X] I want to train my custom dataset, and I have read the [tutorials for training your custom data](https://github.com/meituan/YOLOv6/blob/main/docs/Train_custom_data.md) carefully and organize my dataset correctly; (FYI: We recommand you to apply the config files of xx_finetune.py.) 我想训练自定义数据集，我已经仔细阅读了训练自定义数据的教程，以及按照正确的目录结构存放数据集。（FYI: 我们推荐使用xx_finetune.py等配置文件训练自定义数据集。）

- [X] I have pulled the latest code of main branch to run again and the problem still existed. 我已经拉取了主分支上最新的代码，重新运行之后，问题仍不能解决。

### Search before asking

- [X] I have searched the YOLOv6 [issues](https://github.com/meituan/YOLOv6/issues) and found no similar questions.

### Question

在进行OAT量化时，教师模型的权重teacher_model_path ./assets/yolov6s_v2_reopt_43.1.pt ，这个是哪一步得出的结果？
CUDA_VISIBLE_DEVICES=0,1,2,3,4,5,6,7 python -m torch.distributed.launch --nproc_per_node=8 \
       tools/train.py \
       --data ./data/coco.yaml \
       --output-dir ./runs/opt_train_v6s_qat  \
       --conf configs/repopt/yolov6s_opt_qat.py \
       --quant \
       --distill \
       --distill_feat \
       --batch 128 \
       --epochs 10 \
       --workers 32 \
       --teacher_model_path ./assets/**yolov6s_v2_reopt_43.1.pt** \   这一步的权重是哪里得出的结果
       --device 0,1,2,3,4,5,6,7

**Comment by lanRHC** [comment]
# YOLOv6n model
model = dict(
    type='YOLOv6n',
    pretrained='./assets/v6s_n.pt',
    scales='./assets/v6n_v2_scale_last.pt',
    depth_multiple=0.33,
    width_multiple=0.25,
    backbone=dict(
        type='EfficientRep',
        num_repeats=[1, 6, 12, 18, 6],
        out_channels=[64, 128, 256, 512, 1024],
        ),
    neck=dict(
        type='RepPANNeck',
        num_repeats=[12, 12, 12, 12],
        out_channels=[256, 128, 128, 256, 256, 512],
        ),
    head=dict(
        type='EffiDeHead',
        in_channels=[128, 256, 512],
        num_layers=3,
        begin_indices=24,
        anchors=1,
        out_indices=[17, 20, 23],
        strides=[8, 16, 32],
        iou_type='siou',
        use_dfl=False,
        reg_max=0, #if use_dfl is False, please set reg_max to 0
        distill_weight={
            'class': 1.0,
            'dfl': 1.0,
        },
    )
)

solver = dict(
    optim='SGD',
    lr_scheduler='Cosine',
    lr0=0.00001, #0.01 # 0.02
    lrf=0.001,
    momentum=0.937,
    weight_decay=0.00005,
    warmup_epochs=3.0,
    warmup_momentum=0.8,
    warmup_bias_lr=0.1
)

data_aug = dict(
    hsv_h=0.015,
    hsv_s=0.7,
    hsv_v=0.4,
    degrees=0.0,
    translate=0.1,
    scale=0.5,
    shear=0.0,
    flipud=0.0,
    fliplr=0.5,
    mosaic=1.0,
    mixup=0.0,
)

ptq = dict(
    num_bits = 8,
    calib_batches = 4,
    # 'max', 'histogram'
    calib_method = 'max',
    # 'entropy', 'percentile', 'mse'
    histogram_amax_method='entropy',
    histogram_amax_percentile=99.99,
    calib_output_path='./',
    sensitive_layers_skip=False,
    sensitive_layers_list=[],
)

qat = dict(
    calib_pt = './assets/v6s_n_calib_max.pt',
    sensitive_layers_skip = False,
    sensitive_layers_list=[],
)
# Choose Rep-block by the training Mode, choices=["repvgg", "hyper-search", "repopt"]
training_mode='repopt'

想请问一下大佬们，这个qat里面的 calib_pt文件是哪一步得来的，还是说有提供训练好的权重

**Comment by xingyueye** [comment]
repopt第二步获得的浮点模型

**Comment by lanRHC** [comment]
谢谢解答，另外我还想再问一下，就是在进行PTQ训练时，已经使用了yolov6s_opt_qat.py这个文件，但是这个文件里面的calib_pt如下：
qat = dict(
    calib_pt = './assets/v6s_n_calib_max.pt',
    sensitive_layers_skip = False,
    sensitive_layers_list=[],
)
# Choose Rep-block by the training Mode, choices=["repvgg", "hyper-search", "repopt"]
training_mode='repopt'
我的理解是，这个calib_pt不是应该经过PTQ之后才会生成的吗？在进行PTQ训练时这里应该填什么？希望能够解惑，非常感谢

**Comment by lanRHC** [comment]
> repopt第二步获得的浮点模型

谢谢解答，另外我还想再问一下，就是在进行PTQ训练时，已经使用了yolov6s_opt_qat.py这个文件，但是这个文件里面的calib_pt如下：
qat = dict(
calib_pt = './assets/v6s_n_calib_max.pt',
sensitive_layers_skip = False,
sensitive_layers_list=[],
)

Choose Rep-block by the training Mode, choices=["repvgg", "hyper-search", "repopt"]
training_mode='repopt'
我的理解是，这个calib_pt不是应该经过PTQ之后才会生成的吗？在进行PTQ训练时这里应该填什么？希望能够解惑，非常感谢
